assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -99,6 +101,22 @@ const AdUnits = ( {
 					</div>
 				</div>
 			) }
+			{ serviceData.error && <Notice noticeText={ serviceData.error } isError /> }
+			{ serviceData.created_targeting_keys?.length > 0 && (
+				<Notice
+					noticeText={ [
+						__( 'Created custom targeting keys:' ) + '\u00A0',
+						serviceData.created_targeting_keys.join( ', ' ) + '. \u00A0',
+						<ExternalLink
+							href={ `https://admanager.google.com/${ serviceData.network_code }#inventory/custom_targeting/list` }
+							key="google-ad-manager-custom-targeting-link"
+						>
+							{ __( 'Visit your GAM dashboard' ) }
+						</ExternalLink>,
```

Comment:
Nit: code repetition (also in `assets/wizards/advertising/views/services/index.js`), but we can live with that.